src/workloads/issues/ConnectionsBuildup.yml

```diff
@@ -1,8 +1,10 @@
 SchemaVersion: 2018-07-01
 Owner: "@mongodb/product-perf"
 Description: |
-  This workload created to reproduce SERVER-53853: Large buildup of mongos to mongod connections and
-  low performance with secondaryPreferred reads
+  This workload was created to reproduce SERVER-53853: Large buildup of mongos to mongod connections and
+  low performance with secondaryPreferred reads. For running both with and without sharding enabled.
+  Note that this test was originally two separate files, ConnectionsBuildup.yml and ConnectionsBuildupNoSharding.yml,
+  but was merged into one as part of PERF-4347.
```

Comment:
What's the important metric for this workload ... OperationThroughput?

We should make sure this workload is run on the invisible sharding variant whenever it gets added.

Reply:
The OperationThroughput in the ConnectionsBuildup phase in the main metric we care about. Since this test is short and pretty clear in its intentions I haven't explicitly called this out but if you think that's important I can add a line to the description. Also I notice you've left a comment to include this workload in the invisible sharding variants as part of [PERF-4769](https://jira.mongodb.org/browse/PERF-4769) - thank you!